modules/ROOT/pages/administration/databases.adoc

```diff
@@ -903,6 +926,119 @@ ALTER DATABASE nonExisting IF EXISTS SET TOPOLOGY 1 PRIMARY 0 SECONDARY
 [role="statsonlyqueryresult"]
 0 rows
 
+[role=enterprise-edition]
+[[administration-databases-alter-database-options]]
+=== Options
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE DATABASE `options-example`;
+----
+////
+
+The `ALTER DATABASE` command can be used to set or remove specific options, which are listed below.
+This is a subset of the options allowed for `CREATE DATABASE` as the remaining create options are only applied during the creation of the database.
```

Comment:
```suggestion
```
I don't think this adds any information for the regular user and I'd just take it away.

Reply:
It comes from one of my review comments. To me if feels strange that there are different things I can add to my create with options and my alter with options, I'd have expected to be able to set the same things in both. This sentence (or some variation of it) does point out to me that it is by design that they are different and that we didn't just miss things for ALTER (or when documenting alter)

Reply:
Why would you expect to be able to use on ALTER any other of the options that are available on CREATE? All the other options have to do with seeding the db, or what to do with existing data when the db is first created. None of these would make sense to use on ALTER. 

Whether the options allowed on ALTER are a subset of CREATE or a set with null intersection is un-important to users: either they are the _same_, or they are _different_. Then how much different, and in what way, is not important to the reader. If somebody comes and shouts at us why they can't provide seeding options on ALTER as they can on CREATE, either they don't understood what seeding a db means, or we have failed in explaining it properly.

Reply:
Removed that line, I think the table just following this paragraph already makes it clear about what options are supported.